lib/services/ngsi/entities-NGSI-v2.js

```diff
@@ -521,7 +521,7 @@ function sendUpdateValueNgsi2(entityName, originMeasures, originTypeInformation,
         if (typeof typeInformation.explicitAttrs === 'string') {
```

Comment:
If at the end this PR is going to be merged, an entry in CNR should be include.

In addition, let's check if documentation is properly describing this behaviour (in negative case, it should be modified)

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1618/commits/3176f6cf595a88350a0b25ecd1097fb14c2fe18f